backend/drm/drm.c

```diff
@@ -1150,6 +1160,19 @@ static void page_flip_handler(int fd, unsigned seq,
 		post_drm_surface(&conn->crtc->primary->mgpu_surf);
 	}
 
+	struct timespec present_time = {
+		.tv_sec = tv_sec,
+		.tv_nsec = tv_usec * 1000,
+	};
+	struct wlr_output_event_present present_event = {
```

Comment:
shouldn't `output` also be set?

Reply:
So this part is a little misleading. It's set in `wlr_output_send_present`. I just didn't want to create another struct. Maybe I can just set it here too, if that matters…

Reply:
oh, yeah, I've missed that. Idk if something should be changed then ...